localstack/config.py

```diff
@@ -1046,12 +1044,13 @@ def use_custom_dns():
 # if `DISABLE_BOTO_RETRIES=1` is set, all our created boto clients will have retries disabled
 DISABLE_BOTO_RETRIES = is_env_true("DISABLE_BOTO_RETRIES")
 
-# HINT: Please add deprecated environment variables to deprecations.py
-
 # List of environment variable names used for configuration that are passed from the host into the LocalStack container.
-# Make sure to keep this in sync with the above!
-# Do *not* include any internal developer configurations that apply to host-mode only in this list.
-# Note: do *not* include DATA_DIR in this list, as it is treated separately
+# => Synchronize this list with the above and the configuration docs:
+# https://docs.localstack.cloud/references/configuration/
+# => Sort this list alphabetically
```

Comment:
good  hint! definitely a piece of knowledge that was mostly just lived through tradition but it's good to have this written down here